Makefile

```diff
@@ -130,14 +130,41 @@ oc-clean: oc-down
 	$(PREFIX) rm -rf $(OC_DATA_DIR)
 
 user:
-	$(PYTHON) $(PYDIR)/manage.py createsuperuser --settings=config.settings.local
+	http post localhost:8080/auth/users/create/ \
+	    username=user password=password
+	$(PYTHON) $(PYDIR)/manage.py drf_create_token user --settings=config.settings.local
```

Comment:
Same as the `oc-user`, why not just call login as per `https://github.com/cloudigrade/cloudigrade/blob/master/docs/rest-api-examples.rst#login-to-cloudigrade`?

Reply:
I did it this way for consistency with the other commands, and also because `drf_create_token` has less noise in the output than the HTTP approach.

Reply:
I'd prefer it if we had a clean split, if we're using the api to create a user, we should fetch the token the same way.

Reply:
This concerns me because it's increasing the number of ways we do things, which means more ways things can go wrong.

How about standardizing on the API in all cases? That also means we dogfood our own APIs a bit more, which is always helpful.

Reply:
The auth api is our standard. The token cli command is built into drf, and something that certainly will not be available in production, so we may as well get used to using the method that actual users would use.

Reply:
My mistake, I forgot that there was another reason. We can't use the standard login API because our users have no password. We have to use drf_create_token as a way around that.

Do you think it makes more sense to stick with drf_create_token, or create users with passwords?

Reply:
I've been looking into creating users with passwords, but you really need Python for that. Given our discussion about keeping it simple, I'll just revert to drf_create_token.

Reply:
To be clear, the regular users have passwords, but the superusers do not.

Making the retrieve token methods different would make life *really* difficult for anyone who wanted to build automation around these commands, though, so I strongly think they should work the same way.